wordpress.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: wordpress
+  version: 6.5.4
+  epoch: 0
+  description: "The Open Source Publishing Platform"
+  copyright:
+    - license: GPL-2.0
+  dependencies:
+    provides:
+      - wordpress=${{package.full-version}}
+    runtime:
+      - php
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - git
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/wordpress/wordpress
+      tag: ${{package.version}}
+      expected-commit: e67e9caef43512751aae60f37d91cf589dce78b0
+      destination: "${{targets.destdir}}/usr/share/wordpress"
+
+subpackages:
+  - name: wordpress-config
+    description: Creates WordPress config subpackage that can be overwritten by users.
+    dependencies:
+      runtime:
+        - wordpress
+      provides:
+        - wordpress-config=${{package.full-version}}
```

Comment:
```suggestion
```

This isn't needed. I'm not sure why it would cause a problem, but let's find out.